tapa/graphir_conversion/BUILD.bazel

```diff
@@ -0,0 +1,24 @@
+"""TAPA graphir graphir conversion."""
+
+# Copyright (c) 2025 RapidStream Design Automation, Inc. and contributors.
+# All rights reserved. The contributor(s) of this file has/have agreed to the
+# RapidStream Contributor License Agreement.
+
+load("@rules_python//python:defs.bzl", "py_library")
+
+py_library(
+    name = "graphir_conversion",
+    srcs = glob([
+        "**/*.py",
+    ]),
+    visibility = [
+        "//tapa:__subpackages__",
+    ],
+    deps = [
+        "//tapa:task",
+        "//tapa_graphir",
+        requirement(
+            "pyverilog",
+        ),
```

Comment:
Could you put `requirement("pyverilog"),` in a single line? It cannot take multiple dependencies in a single `requirement` and it's more consistent with other deps in a single line.